### PR TITLE
Remove methods that override identical methods in the superclass

### PR DESCRIPTION
**Pull request by elharo**
## Description
Remove duplicate code

## Motivation and Context
Inheritance FTW

## Impact
none

## Test Plan
CI

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by tdcmeehan** [comment]
@elharo how did you identify these code changes?

**Comment by elharo** [comment]
IntelliJ flagged them all.

File: presto-local-file/src/main/java/com/facebook/presto/localfile/LocalFileMetadata.java [hunk 1]
```diff
@@ -150,12 +148,6 @@ public List<SchemaTableName> listViews(ConnectorSession session, String schemaNa
         return emptyList();
     }
 
-    @Override
```
Comment (rschlussel) [inline presto-local-file/src/main/java/com/facebook/presto/localfile/LocalFileMetadata.java:153]: i think listViews above this is also duplicate.

File: presto-elasticsearch/src/test/java/com/facebook/presto/elasticsearch/TestingElasticsearchConnectorFactory.java [hunk 2]
```diff
@@ -13,62 +13,10 @@
...
-import com.facebook.presto.spi.NodeManager;
-import com.facebook.presto.spi.connector.Connector;
-import com.facebook.presto.spi.connector.ConnectorContext;
-import com.google.inject.Injector;
-
-import java.util.Map;
-
-import static java.util.Objects.requireNonNull;
-
 class TestingElasticsearchConnectorFactory
```
[the remark this answers is not in the document]
Comment (elharo) [reply presto-elasticsearch/src/test/java/com/facebook/presto/elasticsearch/TestingElasticsearchConnectorFactory.java]: done

File: presto-local-file/src/main/java/com/facebook/presto/localfile/LocalFileMetadata.java (hunk 1 above)
Comment (elharo) [reply presto-local-file/src/main/java/com/facebook/presto/localfile/LocalFileMetadata.java:153]: done

**Review by rschlussel — APPROVED** [review]

**Comment by 7c00** [comment]
> IntelliJ flagged them all.

@elharo Do we have any tool, especially maven plugin, to find these code automatically?

**Pull request merged**